ddtrace/ci_visibility/__init__.py

```diff
@@ -0,0 +1,4 @@
+from ddtrace.ci_visibility._recorder import CIVisibility
+
+
+__all__ = ["CIVisibility"]
```

Comment:
Should we make `CIVisibility` an internal component? It is enabled by the pytest and pytestbdd plugins and not by users.

If we decide to make this component public we should add documentation

Reply:
By this, do you mean moving the `ci_visibility` module to `ddtrace/internal/ci_visibility` module? Or just making the `CIVisibility` a private class?

Reply:
Either works